lib/Faker/Plugin/InternetPort.pm

```diff
@@ -0,0 +1,132 @@
+package Faker::Plugin::InternetPort;
+
+use 5.018;
+
+use strict;
+use warnings;
+
+use Venus::Class 'base';
+
+base 'Faker::Plugin';
+
+# METHODS
+
+sub execute {
+  my ($self, $data) = @_;
+
+  return $self->faker->random->bit
+    ? $self->faker->InternetPortDynamic
+    : ( $self->faker->random->bit
+	   ? $self->faker->InternetPortRegistered
+		: $self->faker->InternetPortWellKnown)
+}
```

Comment:
This might be better implemented as:

```perl
sub execute {
  my ($self, $data) = @_;

  my $plugin = $self->faker->random->select([
    'internet_port_dynamic',
    'internet_port_registered',
    'internet_port_well_known',
  ]);

  return $self->faker->$plugin;
}
```